python/py-gwpy/Portfile

```diff
@@ -22,45 +22,44 @@ homepage            https://gwpy.github.io
 master_sites        pypi:g/gwpy \
                     https://github.com/gwpy/gwpy/releases/v${version}/
 
-checksums   rmd160  21a6f0bfba4d7f38359c084f1670de84cb3f8ac9 \
-            sha256  d7f4771eb4b8a794da58779dc01da19a9e4f0f5181923966547e3dcdd957decd \
-            size    1097963
+checksums   rmd160  7ef4e4697558c0a17a1fa45faa1fb21fffa0294e \
+            sha256  47a17967508f006cee9c1c4a62cceff4d6d9870096cad9eedf9f1a4af788ba17 \
+            size    1166915
 
-# until glue is available for multiple python versions, we are stuck to
-# python 2.7
-
-python.versions     27
+python.versions     27 36
 python.default_version 27
 
 if {${name} ne ${subport}} {
 
+    depends_build-append port:py${python.version}-setuptools
+
     depends_lib-append  port:py${python.version}-six \
                         port:py${python.version}-dateutil \
                         port:py${python.version}-numpy \
                         port:py${python.version}-scipy \
                         port:py${python.version}-matplotlib \
                         port:py${python.version}-astropy \
+                        port:py${python.version}-h5py \
                         port:py${python.version}-lal \
-                        port:glue
+                        port:py${python.version}-ligo-segments \
+                        port:py${python.version}-tqdm
+
     if {${python.version} < 34} {
         depends_lib-append port:py${python.version}-enum34
     }
 
-    depends_build-append port:py${python.version}-setuptools
+    if {${python.version} == 27} {
```

Comment:
You are only making the variants available under 2.7?

Reply:
@pmetzger, the extra dependencies included in the variants are only available (currently) for 2.7. If they were available more widely, I would include the variants for 3.6

Reply:
Gotcha.